.github/workflows/publish.yml

```diff
@@ -0,0 +1,52 @@
+name: Publish Python distributions to PyPI and TestPyPI
+on: push # Runs your workflow when you push a commit or tag.
+# on:
+#   release:
+#     types: [published] # triggered whenever a new GitHub release is published
+jobs:
+  build-n-publish:
+    name: Build and publish Python distributions to PyPI and TestPyPI
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@main
+      with:
+          fetch-depth: 0
+    - name: Set up Python 3.10
+      uses: actions/setup-python@v3
+      with:
+        python-version: "3.10"
+    - name: Install pypa/build
+      run: >-
+        python -m
+        pip install
+        build
+        --user
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install pytest
+        if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+    - name: Run unit tests
+      run: >-
+        python -m unittest
+    - name: Build a binary wheel and a source tarball
+      run: >-
+        python -m
+        build
+        --sdist
+        --wheel
+        --outdir dist/
+        .
+    # - name: Publish distribution to Test PyPI
```

Comment:
I don't currently have maintainer access to TestPyPI DeltaCAT [access to ](https://test.pypi.org/project/deltacat/)

Reply:
Maybe we don't need this right now.

Reply:
Ok - removing this commented out block then